OpenRA.Mods.Common/Activities/Air/ReturnToBase.cs

```diff
@@ -147,28 +188,47 @@ public override Activity Tick(Actor self)
 				}
 			}
 
+			var exit = dest.FirstExitOrDefault(null);
+			var offset = exit != null ? exit.Info.SpawnOffset : WVec.Zero;
+
 			List<Activity> landingProcedures = new List<Activity>();
```

Comment:
Do we actually need to make a list of activities here? I think the activities below can just be added to the childactivity queue instead of making a custom list.

Reply:
> Do we actually need to make a list of activities here?

Not really, that's just a leftover from way back (when this was used to get rid of somewhat messy and less readable SequenceActivities usage).
Fixed.